docs/guides/request_storage.mdx

```diff
@@ -145,4 +145,4 @@ import { purgeDefaultStorages } from 'crawlee';
 await purgeDefaultStorages();
 ```
 
-Calling this function will clean up the default request storage directory (and also the request list stored in default key-value store). This is a shortcut for running (optional) `purge` method on the <ApiLink to="types/interface/StorageClient">`StorageClient`</ApiLink> interface, in other words it will call the `purge` method of the underlying storage implementation we are currently using. In addition, this method will make sure the storage is purged only once for a given execution context, so it is safe to call it multiple times.
+Calling this function will clean up the default request storage directory (and also the request list stored in default key-value store). This is a shortcut for running (optional) `purge` method on the <ApiLink to="core/interface/StorageClient">`StorageClient`</ApiLink> interface, in other words it will call the `purge` method of the underlying storage implementation we are currently using. In addition, this method will make sure the storage is purged only once for a given execution context, so it is safe to call it multiple times.
```

Comment:
just curious, the link was not working before? because it was valid, i just added reexport yesterday to the core package. i guess it again (wrongly) marked the one in types packages as reexport? we should probably report this to typedoc, i'd say there will be some issue about this already

Reply:
Yeah, with `types/interface/StorageClient` it returned `Page does not exist` or something like that :/

And yeah - in `types` it has it only in `References` as `Re-exports [StorageClient](https://crawlee.dev/api/core/interface/StorageClient)`